packages/core/src/lib/createSystem.ts

```diff
@@ -66,6 +67,34 @@ function getSudoGraphQLSchema(config: KeystoneConfig) {
   // return createGraphQLSchema(transformedConfig, lists, null, true);
 }
 
+function injectNewDefaults(prismaClient: any, lists: Record<string, InitialisedList>) {
+  for (const listKey in lists) {
+    const list = lists[listKey];
+    const { dbField } = list.fields.id;
+
+    if ('default' in dbField && dbField.default?.kind === 'random') {
+      const { bytes, encoding } = dbField.default;
+      prismaClient = prismaClient.$extends({
+        query: {
+          [list.prisma.listKey]: {
+            async create({ model, args, query }: any) {
+              return query({
+                ...args,
+                data: {
+                  ...args.data,
+                  id: randomBytes(bytes).toString(encoding),
```

Comment:
Will this override any value for `id` that is given? Should we have the same behaviour as the other `default` types - if a value is provided use that?